apps/drupal-default/particle_theme/templates/form/views-exposed-form.html.twig

```diff
@@ -0,0 +1,21 @@
+{#
+/**
+ * @file
+ * Theme override for a views exposed form.
+ *
+ * Available variables:
+ * - form: A render element representing the form.
+ *
+ * @see template_preprocess_views_exposed_form()
+ */
+#}
+{% if q is not empty %}
+  {#
+    This ensures that, if clean URLs are off, the 'q' is added first,
+    as a hidden form element, so that it shows up first in the POST URL.
+  #}
+{{ q }}
+{% endif %}
+<div class="form-inline clearfix mb-3 xl:w-1/3">
+  {{ form }}
```

Comment:
So, this does add padding and a width component for all view forms. The width piece is making an assumption that most view forms will only have one or a few fields, which I think would be the case, but if you see a problem with it lmk.